install_sct

```diff
@@ -41,9 +41,30 @@ DATA_DIR="data"
 PYTHON_DIR="python"
 BIN_DIR="bin"
 
+function die {
+    echo -e "\n\033[0;31m$1\033[0m\n"
+    exit 1
+}
+
+function check_requirements {
+
+  if [[ ! $( which curl ) && ! $( which wget ) ]] ; then
+      die "You need to install \"curl\" OR \"wget\" before you can install the SPINALCORDTOOLBOX \n
+run \n
+> sudo apt-get install wget \n
```

Comment:
the message needs to be distros-agnostic otherwise this command with fail on redhat, osx, etc.

Reply:
You're completely right, I actually left it as it was, but I'll change it. Nice catch !